client/app/dashboard/dashboardController.js

```diff
@@ -0,0 +1,21 @@
+angular.module('client-recon.dashboard', [])
+
+.controller('DashboardController', function ($scope, ClientsApi) {
+  // Your code here
```

Comment:
You can avoid ambiguity by setting the context ($scope) to a var.

Example `var ctrl = this;`

This allow us to maintain lexical scope within functional programming.